gluoncv/auto/estimators/image_classification/image_classification.py

```diff
@@ -224,30 +230,45 @@ def _train_loop(self, train_data, val_data, time_limit=math.inf):
 
             post_tic = time.time()
             train_metric_name, train_metric_score = train_metric.get()
+            if train_metric_name == 'rmse':
+                train_metric_score = - train_metric_score
```

Comment:
Should we add the negation?

Reply:
I think the negation here is to accommodate the checkpoint saving logic, currently the larger metric is preferred, without substituting the callback for model performance comparison, the negation is needed. Maybe we need a better way to handle this.

Reply:
It should be acceptable then.